### PR TITLE
LESS : Use BEM + general improvements

### PR DESCRIPTION
**Pull request by moeenio**
I improved a lot of stuff :

- Improve variables :
  - Add primary-color
  - Rename tertiary to accent
  - Use more logical names
  - Change some colors (I took them from the Figma)
- Do not use absolute/relative positioning, instead use padding/margin
- Use more flexboxes
- Group top-grid, signedin and account-switcher into one accountbar file

### CODE REVIEW

**Review by micahlt — APPROVED** [review]
Looks good to me!  :+1: Thanks a lot!  We'll wait on @Hexsphere's review before merging.

**Comment by ethanhanderson** [comment]
Do I have to review this? 😛

**Review by ethanhanderson — APPROVED** [review]

**Comment by micahlt** [comment]
Approved and merged!

**Pull request merged**
